data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1684 @@
+				goto questions
+			label pay
+			`	The rest of the table look to Lobmab at the same time, prompting her explanation. "Amassed great fortune over our history, House Bebliss has. As part of our gracious help to the Lunarium, providing nearly everything in financial terms we have been."`
+			`	She stops, with the others still looking at her, as if expecting something more. She remains silent, so Chiree looks back at you, continuing, "Also taken some loans from House Plumtab in our stead, Houses Bebliss and Idriss have. Not terribly passionate for our cause, Plumtab's bankers are, but share our interest in booting the Heliarchs, they do, even if meet or talk with us, they never do."`
+				goto questions
+			label task
+			`	Tummug goes around the table, handing copies of some document of sorts to the other leaders, before answering. "One of the center points of Heliarch propaganda, the fear of the Quarg is. Served to justify their budget, and their obsession with hoarding the military, for millennia it has. Thought up a way to mitigate the effect that has on the populace, Oobat has."`
+			`	As if given her cue, Oobat goes further into the plan. "Seen in the Coalition in over six thousand years, no Quarg ship has been. Most likely moved on from our war, they have, unlike the Heliarchs. If proof of the Quarg's lack of interest in waging war against us, we had, spread it to the people, we could. Given the Captain's resourcefulness, and jump drive, thought we did that the best way to acquire that proof, meeting up with the Quarg is." She grabs one of the documents, pointing to it. "For you all to review these questions, I would like. Careful we should be, about how worded they are. Once done, we are, board Captain <last>'s ship, I will. Meet with the Quarg in human space, we must," she finishes as she says the last part looking at you.`
+			`	The Lunarium leaders spend an hour or so going over the questions proposed for the interview, deciding to ditch some, and adding a few new ones. All the while, Elirom asks in a worried manner if Oobat is sure of this idea. "Forgive me if sounding like a Heliarch, I am, but dangerous to leave, it is. How the Quarg would react to seeing someone from the Coalition in their territory, we do not know."`
+			`	"Understand that, I do, but fine I will be. Unlike the Quarg it is, to attack unprovoked," she answers. "And, if worse comes to worse, trust I do in Captain <last>'s abilities to ensure we escape." The leaders go over the interview questions a while longer, until they've reached a consensus as to its completeness. They wish you and Oobat good luck, and you two are escorted back out of the complex, and to your ship. There, Oobat asks you for the best place for such an interview in human space, and you point to <destination>, where they are constructing their ringworld. "Then head there, we must. In your care I will be, Captain." You show her to her bunk, and plot a course to <planet>.`
```

Comment:
The Coalition was the one to initiate the war, not the Quarg. If one can look past the propaganda and find the real history on what happened, it's easy to tell the Quarg weren't going out of their way to attack Coalition ships. In terms of ship combat, they'd know the Quarg weren't enthusiastic about fighting.

A more well known part of each species' history is the first contact with the Quarg (as in face-to-face), which would contain mentions of the Quarg's physique, and the calm manner of both their movement and general demeanor. Only the Heliarchs know of the Quarg who fought tooth and nail to defend the rings, in ground combat; the general Coalition population, and the Lunarium, wouldn't know of that part about the Quarg. Elirom does try to warn against going in the paragraph above, having been a Heliarch, but he's the one member there who thinks it might be dangerous.